tests/storage/conftest.py

```diff
@@ -154,10 +155,8 @@ def tmp_storage(monkeypatch, tmpdir):
     ids=str,
 )
 def tmp_mount(request):
-    mount = request.param
-    if not mount.exists():
-        pytest.xfail("{} storage not available".format(mount.name))
-    return mount
+    with Backend(request.param) as backend:
+        yield backend
```

Comment:
This is fine, but the previous patch is broken without this. We cannot commit broken
patches fixed by the next patch. The backend wrapper should be added first and when the code is ready we can replace the builtin userstorage with the upstream version.